src/messages/state/messagesReducers.ts

```diff
@@ -1,37 +1,36 @@
 import { ActionType } from 'typesafe-actions';
+import { MessageType } from '../types/messagesTypes';
 
 import * as messagesActions from './messagesActions';
 import messagesInitialState, { MessagesState } from './messagesInitialState';
 
 export type MessagesAction = ActionType<typeof messagesActions>;
 
+let fallbackId = 0;
```

Comment:
I think using `uuid` is slightly overengineering it. We just want that no two messages created without a specific ID get the same id. I've added a fixed prefix to avoid collision.